### PR TITLE
Babel

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,5 +46,17 @@
     "eslint-plugin-react-hooks": "^2.3.0",
     "nodemon": "^2.0.2",
     "prettier": "^1.19.1"
+  },
+  "babel": {
+    "presets": [
+      "@babel/preset-env",
+      "@babel/preset-react"
+    ],
+    "plugins": [
```
Comment (adamweeks) [inline package.json]: Please add
"@babel/plugin-proposal-nullish-coalescing-operator"
"@babel/plugin-proposal-optional-chaining"

File: package.json [hunk 2]
```diff
@@ -46,5 +46,17 @@
...
+      "@babel/plugin-proposal-object-rest-spread",
+      "@babel/plugin-proposal-export-default-from",
```
Comment (adamweeks) [inline package.json]: Can you sort these alphabetically?

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "main": "dist/webexWidgets.cjs.js",
   "module": "dist/webexWidgets.esm.js",
   "scripts": {
+    "start": "NODE_ENV=development nodemon src/index.js --exec babel-node",
```
Comment (lalli-flores) [inline package.json]: I don't think this is how we should do our start script. I would leave this out for a different PR once we agree on the repo structure

File: package.json [hunk 4]
```diff
@@ -22,17 +23,48 @@
...
+    "eslint-config-react": "^1.1.7",
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jest": "^23.6.0",
     "eslint-plugin-jsdoc": "^21.0.0",
     "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-mocha": "^6.2.2",
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.18.3",
     "eslint-plugin-react-hooks": "^1.7.0",
+    "nodemon": "^2.0.2",
```
Comment (lalli-flores) [inline package.json]: Should probably add this dep separately / in a different PR

File: package.json (hunk 3 above)
Comment (akoushke) [reply package.json]: that was to check if babel works. yes we dont need it.

Comment (lalli-flores) [reply package.json]: Let's remove it then 👍

Comment (akoushke) [reply package.json]: yes, I'm working on it right now.
